### PR TITLE
LPS-82671 Content Spaces API

### PR DESCRIPTION
**Pull request by javiergamarra**

### CODE REVIEW

File: modules/apps/headless-apio/content-space/content-space-apio-impl/src/main/java/com/liferay/content/space/apio/internal/architect/router/ContentSpaceNestedCollectionRouter.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+/**
+ * Provides the information necessary to expose the ContentSpace resources
+ * through a web API. The resources are mapped from the internal model {@link
+ * Group}.
+ *
+ * @author Eduardo Perez
+ * @review
+ */
+@Component(immediate = true)
+public class ContentSpaceNestedCollectionRouter implements
```
Comment (eperezg) [inline modules/apps/headless-apio/content-space/content-space-apio-impl/src/main/java/com/liferay/content/space/apio/internal/architect/router/ContentSpaceNestedCollectionRouter.java]: delete this class or expose on the representor as a sub-content-space

File: modules/apps/headless-apio/site/site-apio-impl/src/main/java/com/liferay/site/apio/internal/architect/resource/WebSiteCollectionResource.java [hunk 2]
```diff
@@ -84,16 +84,14 @@ public String getName() {
...
 		).addLinkedModel(
 			"creator", PersonIdentifier.class, Group::getCreatorUserId
-		).addLinkedModel(
-			"folder", RootFolderIdentifier.class, Group::getGroupId
 		).addLocalizedStringByLocale(
 			"description", Group::getDescription
 		).addLocalizedStringByLocale(
 			"name", Group::getName
+		).addLinkedModel(
+			"contentSpace", ContentSpaceIdentifier.class, Group::getGroupId
```
Comment (eperezg) [inline modules/apps/headless-apio/site/site-apio-impl/src/main/java/com/liferay/site/apio/internal/architect/resource/WebSiteCollectionResource.java]: alphabetical order

File: modules/apps/headless-apio/content-space/content-space-apio-impl/src/main/java/com/liferay/content/space/apio/internal/architect/router/ContentSpaceNestedCollectionRouter.java (hunk 1 above)
Comment (javiergamarra) [reply modules/apps/headless-apio/content-space/content-space-apio-impl/src/main/java/com/liferay/content/space/apio/internal/architect/router/ContentSpaceNestedCollectionRouter.java]: Removed :)

File: modules/apps/headless-apio/site/site-apio-impl/src/main/java/com/liferay/site/apio/internal/architect/resource/WebSiteCollectionResource.java (hunk 2 above)
Comment (javiergamarra) [reply modules/apps/headless-apio/site/site-apio-impl/src/main/java/com/liferay/site/apio/internal/architect/resource/WebSiteCollectionResource.java]: Fixed :) https://github.com/eperezg/liferay-portal/pull/27/commits/d876872ce197f8c2886ffd41dffd9f19ac07c7b4

**Comment by javiergamarra** [comment]
Sent here: https://github.com/brianchandotcom/liferay-portal/pull/61010

Thanks :D

**Pull request closed without merging**